tests/unit/module_utils/test_linode_common.py

```diff
@@ -0,0 +1,21 @@
+from types import SimpleNamespace
+import os
+
+from ansible_collections.linode.cloud.tests.unit.base import TestModuleBase
+
+class TestLinodeModuleBase():
```

Comment:
Just a minor optional suggestion, if the test case is not depends on any class/object attributes (`self.some_attr`), we can even remove the classes and only keep the functions, lol